src/librustdoc/html/markdown.rs

```diff
@@ -1293,6 +1284,16 @@ impl LangString {
                     LangStringToken::LangToken(x) if x.starts_with("edition") => {
                         data.edition = x[7..].parse::<Edition>().ok();
                     }
+                    LangStringToken::LangToken(x) if x.starts_with("rust") => {
+                        if x[4..].parse::<Edition>().is_ok()
+                            && let Some(extra) = extra
+                        {
+                            extra.error_invalid_codeblock_attr(format!(
+                                "unknown attribute `{x}`. Did you mean `edition{}`?",
```

Comment:
The rustc-dev-guide says ["did you mean" should be avoided](https://rustc-dev-guide.rust-lang.org/diagnostics.html#suggestion-style-guide):
> Suggestions should not be a question. In particular, language like "did you mean" should be avoided. Sometimes, it's unclear why a particular suggestion is being made. In these cases, it's better to be upfront about what the suggestion is.
>
> Compare "did you mean: Foo" vs. "there is a struct with a similar name: Foo".

Reply:
Oh nice! I'll update all the suggestions then. :)